templates/features.html

```diff
@@ -117,7 +117,10 @@
 var featuresFetchMetric = new Metric('features_loaded');
 
 chromeMetadata.addEventListener('query-changed', function(e) {
-  search.value = '=' + e.detail.version;
+  var value = e.detail.version;
+  var isMilestone = value.match(/^[0-9]+$/);
+  search.value = isMilestone ? 'milestone=' + value :
```

Comment:
Ah sorry. I may have been looking at commit view instead of all changes. I see it now.